plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/AbstractXmlSourceValidatorTest.java

```diff
@@ -150,6 +162,7 @@ public void testGetProject() throws CoreException {
     IProject testProject = AbstractXmlSourceValidator.getProject(helper);
     assertNotNull(testProject);
     assertEquals(project, testProject);
+    file.delete(true, null);
```

Comment:
May want to put this in a finally block so it happens even if an exception is thrown.